### PR TITLE
Nice meta

### PR DESCRIPTION
**Pull request by marcharper**
Once upon a time I made all the MetaWinner strategies "nice" -- they never defect unless the opponent does first. This added a lot of nice strategies to the library. In this PR I add a transformer to make any strategy nice, and diversify the meta strategy collection a bit by making some not nice.

I recommend that we remove some of the variants from the standard collection in the near future-- there's not much difference in performance among the variants, and there are a number of better strategies incoming. But that's a conversation for another PR.

### CODE REVIEW

File: axelrod/strategy_transformers.py [hunk 1]
```diff
@@ -219,6 +216,19 @@ def forgiver_wrapper(player, opponent, action, p):
     forgiver_wrapper, name_prefix="Forgiving")
 
 
+def nice_wrapper(player, opponent, action):
+    """If a strategy wants to defect, flip to cooperate with the given
```
Comment (drvinceknight) [inline axelrod/strategy_transformers.py]: This docstring isn't correct right? There is no probability involved.

Comment (marcharper) [inline axelrod/strategy_transformers.py]: Yep, lazy copy paste on my part.

Comment (marcharper) [inline axelrod/strategy_transformers.py]: Done

**Comment by drvinceknight** [comment]
One final thing, could you add a little something about this transformer to http://axelrod.readthedocs.io/en/latest/tutorials/advanced/strategy_transformers.html?

**Pull request merged**